src/pages/home/getHomePageData.ts

```diff
@@ -0,0 +1,15 @@
+import {CancerStudy, TypeOfCancer} from "../../shared/api/CBioPortalAPI";
+
+export type HomePageData = {cancerTypes:TypeOfCancer[], studies:CancerStudy[]};
+
+function getHomePageData():Promise<HomePageData>

```

Comment:
We should go head and call this something more feature specific than homePageData.